src/lib/write-package.ts

```diff
@@ -118,13 +118,15 @@ export const writePackage = async (
   }
 
   console.info('latest repo ', latest.repository);
-
-  const repo = repoToGithub(latest.repository);
+  //tslint:disable-next-line:no-any
```

Comment:
The format for eslint here is different, which is why you're still getting the linter warning.  
```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Reply:
ah thanks i gave up after googling and stiff drink